libs/tree-sitter-wing/grammar.js

```diff
@@ -617,36 +643,28 @@ module.exports = grammar({
         field("parameter_list", $.parameter_list),
         optional($._return_type),
         "=>",
-        field("block", $.block)
+        field("block", $.block),
       ),
 
     await_expression: ($) => prec.right(seq("await", $.expression)),
     defer_expression: ($) => prec.right(seq("defer", $.expression)),
     parenthesized_expression: ($) => seq("(", $.expression, ")"),
 
-    _collection_literal: ($) =>
-      choice($.array_literal, $.set_literal, $.map_literal),
-    array_literal: ($) =>
-      seq(
-        optional(field("type", $._builtin_container_type)),
-        "[",
-        commaSep(field("element", $.expression)),
-        "]"
-      ),
-    set_literal: ($) =>
+    _collection_literal: ($) => choice($.array_or_set_literal, $.map_literal),
+    array_or_set_literal: ($) =>
```

Comment:
nit: I would just keep this as array_literal, conceptually we don't really have a set literal anymore

Reply:
interesting; I did this rename based on https://github.com/winglang/wing/issues/4843#issuecomment-1866956662

but you have a point that there's not set literal anymore in the codebase